### DISCUSSION TITLE
reset properties not working if i use vite tailwind after submit in livewire 3

### DISCUSSION THREAD

**Discussion by restu-lomboe** [livewire/livewire#6805] [1. Help] [answered]
this is mw livewire component
```
<?php

namespace App\Livewire;

use Livewire\Component;

class About extends Component
{
    public $chat = '';

    public function save(){
        $this->reset();
    }

    public function render()
    {
        return view('livewire.about');
    }
}
```

this is my html
```
<!DOCTYPE html>
<html lang="{{ str_replace('_', '-', app()->getLocale()) }}">
    <head>
        <meta charset="utf-8">
        <meta name="viewport" content="width=device-width, initial-scale=1">

        <title>Laravel</title>

        <!-- Fonts -->
        <link rel="preconnect" href="https://fonts.bunny.net">
        <link href="https://fonts.bunny.net/css?family=figtree:400,600&display=swap" rel="stylesheet" />

        <!-- Scripts -->
        @vite(['resources/css/app.css', 'resources/js/app.js'])
    </head>
    <body>
        @livewire('about')
    </body>
</html>
```

this is my livewire blade
```
<div>
    <form wire:submit.prevent="save">
        <input type="text" wire:model="chat">
        <button>submit</button>
    </form>
 </div>
```

i have already running
npm run dev
php artisan serve

but if i remove this from head html @vite(['resources/css/app.css', 'resources/js/app.js']) 
$this->reset() working

**Comment by joshhanley** [comment]
@restu-lomboe do you have any browser console errors?

**Reply by restu-lomboe** [reply]
No
it's normal

**Reply by joshhanley** [reply]
@restu-lomboe what does your app.js file look like?

**Reply by restu-lomboe** [reply]
```
import './bootstrap';

import Alpine from 'alpinejs';

window.Alpine = Alpine;

Alpine.start();
```

like this

**Reply by joshhanley** [reply, marked as answer]
@restu-lomboe Livewire V3 now includes Alpine bundled in, so you will need to remove Alpine from you `app.js`.

**Reply by restu-lomboe** [reply]
solved bro,
i remove apline from app.js

thanks
you save my day

**Reply by joshhanley** [reply]
@restu-lomboe no worries! 🙂

**Reply by Mushkir** [reply]
@joshhanley Hey man! Thank you so much :). You really save my day....

**Comment by Aliy7** [comment]
I have similar issues removed alpine imports but still not working 
import { document } from 'postcss';
import './bootstrap';

// import Alpine from 'alpinejs';



// window.Alpine = Alpine;

// Alpine.start();
document.addEventListeners('DomContentLoaded', () =>{
  console.log("dom loaded")
});
document.addEventListeners('livewire:navigated', () =>{
    console.log("Navigator");
    initFlowbite();
  });

**Comment by Aliy7** [comment]
I used this to solve post not clearing issues following this guidance https://stackoverflow.com/questions/76529809/vite-resources-css-app-css-resources-js-app-js-is-not-working-with-larav
